### DISCUSSION TITLE
Showcase: Built something cool?

### DISCUSSION THREAD

**Discussion by janosh** [janosh/svelte-multiselect#200] [Show and tell]
If you've built something cool 😎  with `svelte-multiselect`, this is the place to post it 📮 to inspire others or solicit feedback 🎉 . Here's an example that uses Multiselect to switch between different heatmaps on a periodic table:

https://sveriodic-table.janosh.dev

**Comment by janosh** [comment]
This looks very useful: [scambier/obsidian-text-extractor](https://github.com/scambier/obsidian-text-extractor). OCR right in your [markdown knowledge base](https://obsidian.md). Shout out to @scambier.

![context_menu](https://user-images.githubusercontent.com/30958850/216499707-4450096e-5708-42e6-8f3b-7277463028e1.png)

**Comment by scambier** [comment]
I'm using your package in the settings page, to select the OCR languages :)

![image](https://user-images.githubusercontent.com/3216752/216534000-406ed7d7-8571-47ec-8726-5828460daa48.png)

**Reply by janosh** [reply]
Nice! Looks good!

**Comment by janosh** [comment]
Not sure if officially affiliated with Effective Altruism but someone made a nice web UI over at https://openbook.fyi to see
how many donations the most effective charities (according to [GiveWell](https://givewell.org)) have received.

https://github.com/rachelweinberg12/OpenBook
